photonjs/README.md

```diff
@@ -161,6 +162,44 @@ type Profile = {
   - Terminating/chainable operation
   - Graph selection: Travesal vs operate
   - ...
+  
+## Debugging
+```ts
+
+// Either like this
+const photon = new Photon({
+  logQueries: 'pipe',
+})
+
+// Or like this
+const photon = new Photon({
+  debug: {
+    queries: 'pipe', // | true | 'inherit', 'inherit' = true
+    engine: 'pipe'
```

Comment:
What's the difference between `queries` and `engine`?

Reply:
Moved everything `engine` related out to `log`.